README.md

```diff
@@ -68,6 +68,24 @@ Just make sure that the version of [react-navigation](https://github.com/react-n
 
 You are all set 🎉 – when screens are enabled in your application code react-navigation will automatically use them instead of relying on plain React Native Views.
 
+### Experimental support for `react-freeze`
+
+> You have to use React Native 0.64 or higher, react-navigation 5.x or 6.x and react-native-screens 3.9.0 or higher
+
+Since `v3.9.0` `react-native-screens` comes with experimental support for `react-freeze`. It uses the React Suspense mechanism to prevent parts of the react component tree from rendering while keeping its state untouched.
+
+To benefit from this feature enable it in your entry file (e.g. `App.js`) with this snippet:
```

Comment:
```suggestion
To benefit from this feature, enable it in your entry file (e.g. `App.js`) with this snippet:
```

Reply:
Fixed in 18ad337084307c2bcebb7b676eab93722f109c4d